crates/matrix-sdk-indexeddb/src/crypto_store/migrations/mod.rs

```diff
@@ -157,6 +163,16 @@ fn add_nonunique_index<'a>(
     object_store.create_index_with_params(name, &IdbKeyPath::str(key_path), &params)
 }
 
+fn add_nonunique_multi_key_index<'a>(
+    object_store: &'a IdbObjectStore<'a>,
+    name: &str,
+    key_path: &[&str],
+) -> Result<IdbIndex<'a>, DomException> {
+    let mut params = IdbIndexParameters::new();
+    params.unique(false);
+    object_store.create_index_with_params(name, &IdbKeyPath::str_sequence(key_path), &params)
```

Comment:
It's not a *strong* opinion, but it's worth noting that this whole thing can equivalently be written:

```suggestion
    object_store.create_index(name, key_path.into())
```

... at which point, I really question whether the helper function has any value.

(even if you want to make the nonunique bit explicit, that's not much longer, because it can be written: `IdbIndexParameters::new().unique(false)`)